### PR TITLE
add pod-checkpointer-operator repo config and jobs

### PR DESCRIPTION
**Pull request by sjenning**
fyi @rphillips

### CODE REVIEW

**Comment by sjenning** [comment]
fyi @derekwaynecarr

**Comment by droslean** [comment]
the ci-operator's config should be named as `openshift-pod-checkpointer-operator-master.yaml`

File: ci-operator/config/openshift/pod-checkpointer-operator/openshift-pod-checkpointer-operator-master.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    bin:
+      as:
+      - "0"
+  to: pod-checkpointer-operator
+
+tag_specification:
+  cluster: https://api.ci.openshift.org
+  name: origin-v4.0
+  namespace: openshift
+  tag: ''
```
Comment (droslean) [inline ci-operator/config/openshift/pod-checkpointer-operator/openshift-pod-checkpointer-operator-master.yaml]: you can remove this.

File: ci-operator/config/openshift/pod-checkpointer-operator/openshift-pod-checkpointer-operator-master.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  tag_overrides: {}
```
Comment (droslean) [inline ci-operator/config/openshift/pod-checkpointer-operator/openshift-pod-checkpointer-operator-master.yaml]: and this

**Pull request merged**